core/native-bridge.ts

```diff
@@ -2,7 +2,7 @@
  * Note: When making changes to this file, run `npm run build:nativebridge`
  * afterwards to build the nativebridge.js files to the android and iOS projects.
  */
-import type { HttpResponse } from './src/core-plugins';
+import { isRelativeOrProxyUrl, type HttpResponse, createProxyUrl } from './src/core-plugins';
```

Comment:
lint is complaining here because it doesn't allow type imports with regular imports, should be separate imports.
But anyway, I think isRelativeOrProxyUrl and createProxyUrl could be both inside native-bridge.ts file, not in core-plugins as those are not plugin methods but helpers.
When I said we could export the createProxyUrl method, I meant as a Capacitor function like `convertFileSrc` function here https://github.com/ionic-team/capacitor/blob/main/core/native-bridge.ts#L865
But that's going to require a few more changes, so we can export it later.